pkgs/development/python-modules/python-tap/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "python-tap";
+  version = "3.0";
+
+  src = fetchFromGitHub {
+    owner = "python-tap";
+    repo = "tappy";
+    rev = "v${version}";
+    sha256 = "sha256-KKuIIG5IRsVIWxFv+amn/fo7CFMkQW3hbCA0ygJROyc=";
+  };
```

Comment:
Missing pythonImportsCheck.

Reply:
Sure. I just tried to satisfy build/test depends of other stuff

Reply:
I fix pythonImportsCheck if folks decide this PR better than other, otherwise I pass it (or do python-tap/tappy as own)